src/commands/convert/msft_secure2hdf.ts

```diff
@@ -0,0 +1,130 @@
+import {Command, Flags} from '@oclif/core'
+import fs from 'fs'
+import {MsftSecureScoreMapper as Mapper} from '@mitre/hdf-converters'
+import {checkSuffix} from '../../utils/global'
+import {ClientSecretCredential} from '@azure/identity'
+import {Client, ClientOptions} from '@microsoft/microsoft-graph-client'
+import {
+  SecureScore,
+  SecureScoreControlProfile,
+} from '@microsoft/microsoft-graph-types'
+import {TokenCredentialAuthenticationProvider} from '@microsoft/microsoft-graph-client/authProviders/azureTokenCredentials'
+
+function processInputs(
+  scoreDoc: SecureScore,
+  profiles: { value: SecureScoreControlProfile[] },
+  output: string,
+  withRaw: boolean,
+): any {
+  const converter = new Mapper(
+    JSON.stringify({
+      secureScore: scoreDoc,
+      profiles: profiles,
+    }),
+    withRaw,
+  )
+  fs.writeFileSync(checkSuffix(output), JSON.stringify(converter.toHdf()))
+}
+
+export default class MsftSecure2HDF extends Command {
+  static usage = [
+    'convert msft_secure2hdf -p <secureScoreProfile-json> -r <secureScore-json> [-h]',
+    'convert msft_secure2hdf -t <azure-tenant-id> -a <app-id> -s <app-secret> [-h]',
+  ];
+
+  static description =
+    'Translate a Microsoft365 Secure Score report to a Heimdall Data Format JSON file';
+
+  static examples = [
```

Comment:
you've only got one example listed here whereas the readme has two.  please use the mapper as the source of truth for this information and copy paste / make minor formatting edits to the `saf convert msftsecure2hdf --help` output for what to put in the readme.  if you want two examples over there, please put two here.